### ISSUE TITLE
Enabling / disabling the laptopinternal display

### ISSUE THREAD

**Issue by t2b3** [swaywm/sway#7978]
E

### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions on IRC at #sway on Libera Chat.
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Please do NOT submit issues for information from the github wiki. The github wiki is community maintained and therefore may contain outdated information, scripts that don't work or obsolete workarounds.
  If you fix a script or find outdated information, don't hesitate to adjust the wiki page.

### Please fill out the following:
- **Sway Version:**
sway version 1.9-rc.3-9c13ed6e (Feb 12 2024, branch 'HEAD')

- **Debug Log:**
  - Run `sway -d 2> ~/sway.log` from a TTY and upload it to a pastebin, such as gist.github.com.
  - This will record information about sway's activity. Please try to keep the reproduction as brief as possible and exit sway.
  - Attach the **full** file, do not truncate it.

- **Configuration File:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.

- **Stack Trace:**
  - This is only needed if sway crashes.
  - If you use systemd, you should be able to open the coredump of the most recent crash with gdb with
    `coredumpctl gdb sway` and then `bt full` to obtain the stack trace.
  - If the lines mentioning sway or wlroots have `??` for the location, your binaries were built without debug symbols. Please compile both sway and wlroots from source and try to reproduce.

- **Description:**
  - The steps you took in plain English to reproduce the problem.

https://github.com/swaywm/sway/assets/3286756/bb4a2e3e-fb99-4c61-b6ed-fbb54bb2a80f

**Comment by emersion** [comment]
Can you reproduce with the latest commit?

Here are some instructions to compile from source: https://github.com/swaywm/sway/wiki/Development-Setup#compiling-as-a-subproject

**Comment by t2b3** [comment]
> Can you reproduce with the latest commit?
> 
> 
> 
> Here are some instructions to compile from source: https://github.com/swaywm/sway/wiki/Development-Setup#compiling-as-a-subproject

Thanks for your reply. I used the branch for the release candidate together with the latest stable wlroots release (as subproject). I will try to test with the latest commits of each project.

**Comment by emersion** [comment]
Also, is this a regression? Are you able to reproduce with sway 1.8, or not?

**Comment by t2b3** [comment]
> Also, is this a regression? Are you able to reproduce with sway 1.8, or not?

This seems to be a regression. It does not happen with sway 1.8.1 and wlroots 0.16.2.

**Comment by t2b3** [comment]
Problems persists with latest commit on sway and wlroots.

**Comment by t2b3** [comment]
In addition to that, a strange overlap occures between the internal display and the first external one.
